### PR TITLE
Product info

### PR DESCRIPTION
**Pull request by jleiandy**
write tests for product overview

fix components that failed product overview test

update code coverage to find all files

add mock data for tests

add test scripts

### CODE REVIEW

File: client/components/ProductOverview/ImageGallery.jsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import '../../css/ImageGallery.scss';
 const ImageGallery = ({
   currentStyle, imageIndex, updateImageIndex, expand,
 }) => {
+  // console.log(currentStyle, imageIndex, updateImageIndex, expand)
```
Comment (cschucode) [inline client/components/ProductOverview/ImageGallery.jsx]: Still need this here?

File: jest.config.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ module.exports = {
 
   // An array of glob patterns indicating a set of files for which coverage information
   // should be collected
-  // collectCoverageFrom: undefined,
+  collectCoverageFrom: ['client/**/*.{js,jsx}'],
```
Comment (cschucode) [inline jest.config.js:24]: I was playing with this as well, but it seemed to do the same thing when it was undefined.

**Review by cschucode — APPROVED** [review]
Nice work on the spec files. 💯   Key takeaways for me:
* mocking axios
* GUI coverage report
* jest modifiers for watching test files during development.

File: client/components/ProductOverview/ImageGallery.jsx (hunk 1 above)
Comment (jleiandy) [reply client/components/ProductOverview/ImageGallery.jsx]: removed

File: jest.config.js (hunk 2 above)
Comment (jleiandy) [reply jest.config.js:24]: oh well, now it is more explicit.

**Pull request merged**